workspaces/graph-lib/src/endpoints-graph/index.ts

```diff
@@ -111,6 +111,36 @@ export type Edge =
 const isNodeRemoved = (node: Node): boolean =>
   node.type !== NodeType.BatchCommit && node.data.isRemoved;
 
+////////////////////////////////////////////////////////////////////////////////
+const getLatestCreatedNodeId = (
```

Comment:
We can use this to also handle requests and responses. I'm planning on handling this tomorrow

Reply:
I also wonder if this should live in graph-lib, or if it should be pulled back into the spectacle/projection/endpoints

The reason I hesitate to pull it back into the spectacle projection is because I don't know if only returning the most recent node generally makes sense, and if we need the old nodes in certain cases.